how-to/configure-celery.rst

```diff
@@ -59,15 +61,15 @@ These tasks are covered in order below.
 Using the broker environment variable
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-For Celery, we provide a ``DEFAULT_AMQP_BROKER_URL`` (in some older applications, simply ``BROKER_URL``). This provides
+For Celery, we provide a ``DEFAULT_AMQP_BROKER_URL`` (in Aldryn celery, simply ``BROKER_URL``). This provides
```

Comment:
Celery with capital C.

Reply:
It's not that we provide a different env var, it's just that Aldryn Celery _uses_ a different one. So you can say 

```
... (the same value is also available as `BROKER_URL`, provided for legacy Aldryn Celery applications).
```